### PR TITLE
Multi brand search form for BFB

### PR DESCRIPTION
**Pull request by jordanhillier90**
#### What does this PR do? (please provide any background)
Changes to navigation and jumbotron to facilitate multiple brands on BFB

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### What gif best describes how you feel about this work?

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: brands/budget-family-breaks/src/less/jumbotron.less [hunk 1]
```diff
@@ -1 +1,5 @@
-@import '../../../brand-globals/src/less/jumbotron';
\ No newline at end of file
+@import '../../../brand-globals/src/less/jumbotron';
+
+.jumbotron.jumbotron-with-form .nav-tabs > li {
```
Comment (nickine) [inline brands/budget-family-breaks/src/less/jumbotron.less]: .jumbotron-with-form seems to be a prevalent thing across brands.  I wonder if we need to think about abstracting it up to Globals? What you reckon @nassimdonald @jodiedoubleday @stuartgiles ??

File: brands/budget-family-breaks/src/less/navs.less [hunk 2]
```diff
@@ -1,21 +1,48 @@
...
+}
+
 .nav-tabs {
-  border-bottom: transparent;
   > li {
     > a {
-      &:hover {
-        color: @brand-primary;
-        border-bottom-color: @nav-tabs-link-hover-border-color;
+      line-height: 1.2;
```
Comment (nickine) [inline brands/budget-family-breaks/src/less/navs.less]: Would `@line-height-base` work instead?

File: brands/budget-family-breaks/src/less/navs.less [hunk 3]
```diff
@@ -1,21 +1,48 @@
...
+      &:hover,
+      &:focus {
+        border-top: 3px solid @brand-primary;
+      }
+    }
+  }
+  &.nav-justified {
+    > li {
+      > a {
+        border: 1px solid #e4e4e4;
```
Comment (nickine) [inline brands/budget-family-breaks/src/less/navs.less]: Would `@nav-tabs-justified-link-border-color` work over a hard coded colour?

File: brands/budget-family-breaks/src/less/jumbotron.less (hunk 1 above)
Comment (nassimdonald) [inline brands/budget-family-breaks/src/less/jumbotron.less]: This actually needs to be refactored before going anywhere near globals. It's not a very flexible component right now, it was written very brand-specific. I'm happy to revisit if there is a need for this in other brands?

**Pull request merged**